src/js/tabs/trust.js

```diff
@@ -466,8 +473,13 @@ TrustTab.prototype.angular = function (module)
           tx.addMemo('client', 'rt' + $scope.version);
 
           tx.trustSet(idAccount, '0' + '/' + lineCurrency + '/' + lineAccount);
-          $scope.acctDefaultRippleFlag ? tx.setFlags('ClearNoRipple') : tx.setFlags('NoRipple');
-
+          var flags = ['ClearFreeze'];
```

Comment:
"The freeze flag must be cleared or the trust line cannot be removed." -- David Schwartz